bundles/org.openhab.binding.wemo/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -354,4 +380,50 @@
 		<description>Date/time the coffee maker last completed cleaning</description>
 	</channel-type>
 
+    <channel-type id="faderCountDownTime" advanced="true">
+        <item-type>Number</item-type>
+        <label>Fader CountDown Time</label>
+        <description>The fading duration time in minutes</description>
+        <state min="1" max="30" step="1.0" pattern="%d min" readOnly="false"/>
+    </channel-type>
+
+     <channel-type id="faderEnabled" advanced="true">
+        <item-type>Switch</item-type>
+        <label>Fader OFF/ON</label>
+        <description>Allows to switch the fader ON/OFF</description>
+    </channel-type>
+    
+    <channel-type id="timerStart" advanced="true">
+       <item-type>Switch</item-type>
+       <label>Timer OFF/ON</label>
+       <description>Allows to switch the timer ON/OFF</description>
+    </channel-type>
+    
+
+     <channel-type id="nightMode" advanced="true">
+        <item-type>Switch</item-type>
+        <label>NightMode OFF/ON</label>
+        <description>Allows to switch the nightMode ON/OFF</description>
+    </channel-type>
+
+     <channel-type id="startTime" advanced="true">
+        <item-type>DateTime</item-type>
+        <label>NightMode Start Time</label>
+        <description>Time when the Night Mode starts</description>
+        <state pattern="%1$tR" readOnly="true"/>
+    </channel-type>
+
+     <channel-type id="endTime" advanced="true">
+        <item-type>DateTime</item-type>
+        <label>NightMode End Time</label>
+        <description>Time when the Night Mode ends</description>
+        <state pattern="%1$tR" readOnly="true"/>
+    </channel-type>
+
+     <channel-type id="nightModeBrightness" advanced="true">

```

Comment:
This should imho as well be a configuration parameter of the `brightness` channel.

Reply:
No, it is independent from normal brigthness.

Reply:
Can you please elaborate? Can you send a command to `nightModeBrightness` and it will immediately do something? But only if it is in nightmode?

Reply:
It will set the brightness for nightmode independent to the brightness for normal dimming. So yes, the device has 2 independent brighness values.

Reply:
Again, this is a configuration and not anything that "does something".
What does trigger the dimmer to go to the configured nightmode brightness? Isn't this triggered by sending ON to the `brightness` channel (if the dimmer is in nightmode and otherwise an `ON` to the very same channel results in the not-nightmode mode)? If my understanding is right, this is clearly a configuration of the `brightness` channel.

Reply:
Sorry, but your understanding is not right. WeMo DimmerSwitch works completely different.
See my comment on fader, its is basically the same for nightMode.

Reply:
So same question as for the fader: Which openHAB channel actively triggers the nightmode brightness?

Reply:
CHANNEL_NIGHTMODE does the switching, CHANNEL_NIGHTMODEBRIGHTNESS sets the brightness.

Reply:
So isn't CHANNEL_NIGHTMODEBRIGHTNESS a configuration of the CHANNEL_NIGHTMODE then?

Reply:
If so, how would one present a channel configuration in the UI ?